app/src/main/java/hu/vmiklos/plees_tracker/calendar/CalendarImport.kt

```diff
@@ -0,0 +1,92 @@
+package hu.vmiklos.plees_tracker.calendar
+
+import android.content.ContentResolver
+import android.content.Context
+import android.net.Uri
+import android.provider.CalendarContract.Calendars
+import android.provider.CalendarContract.Events
+import hu.vmiklos.plees_tracker.Sleep
+
+object CalendarImport {
```

Comment:
Whenever you add a new class, could you always add at least a one-liner description of what it does? As you see, ~all classes in the codebase do this, it would be nice not to break that.